skfda/representation/_functional_data.py

```diff
@@ -788,6 +788,26 @@ def concatenate(self, *others, as_coordinates=False):
         """
         pass
 
+    @staticmethod
+    @abstractmethod
+    def concatenate_samples(objects, as_coordinates=False):
+        """Join samples from a list of similar FData objects.
+
+        Joins samples of FData objects if they have the same dimensions and
+        sampling points.
+
+        Args:
+            objects (list of :obj:`FData`): Objects to be concatenated.
+            as_coordinates (boolean, optional):  If False concatenates as
```

Comment:
It is `concatenate_samples`, so `as_coordinates` should not be an option

Reply:
Parameter removed